src/app.ts

```diff
@@ -91,4 +91,35 @@ app.get('/flights', (req, res) => {
   res.json(flights);
 });
 
+// /airport
+// Retrieve a list of details for a given airport
+app.get('/airport', (req, res) => {
+  const codeFormatText = 'AAA';
+  const { query } = req;
+
+  if (!query || !query.code) {
+    res.status(400).send(`airport 'code' is a required parameter and must use the following format: ${codeFormatText}`);
+    return;
+  }
+  const userCode = query.code.toUpperCase();
```

Comment:
We for sure need this, but let's move it up to where we first get this variable. While we're moving it, let's see if we can make the name slightly more relevant.
```suggestion
```